distribution/kernel/carbon-home/dbscripts/h2.sql

```diff
@@ -284,6 +284,7 @@ CREATE TABLE IF NOT EXISTS UM_PERMISSION (
 			UM_RESOURCE_ID VARCHAR(255) NOT NULL,
 			UM_ACTION VARCHAR(255) NOT NULL,
 			UM_TENANT_ID INTEGER DEFAULT 0,
+			UNIQUE(RESOURCE_ID, ACTION),
```

Comment:
This should be UM_RESOURCE_ID and UM_ACTION.

BTW these two will not be unique.

should be corrected as follows
UNIQUE(UM_RESOURCE_ID,UM_ ACTION, UM_TENANT_ID),